There is no <error.h> in musl libc, thus enable the
include only if __GLIB__ is defined.

--- a/3rdparty/iaxclient/lib/libiax2/src/iax.c	2015-02-18 21:10:05.000000000 +0100
+++ b/3rdparty/iaxclient/lib/libiax2/src/iax.c	2015-10-31 17:14:55.477461723 +0100
@@ -75,7 +75,7 @@
 // FlightGear: Modified to include FreeBSD
 #if !defined(MACOSX) && !defined(__OpenBSD__) && !defined(__FreeBSD__)
 #include <malloc.h>
-#if !defined(SOLARIS)
+#if !defined(SOLARIS) && defined(__GLIBC__)
 #include <error.h>
 #endif
 #endif
